hedera-node/test-clients/src/main/java/com/hedera/services/bdd/utils/Signing.java

```diff
@@ -102,5 +103,12 @@ public static byte[] signMessage(final byte[] messageHash, byte[] privateKey) {
         return result;
     }
 
+    public static byte[] signMessageEd25519(final byte[] message, byte[] privateKey) {
```

Comment:
I don't think this is being used anymore.

Reply:
ok, will remove

Reply:
To be fixed #16448